docs/data-sources/metastore.md

```diff
@@ -0,0 +1,53 @@
+---
+subcategory: "Unity Catalog"
+---
+# databricks_metastore Data Source
+
+Retrieves information about metastore for a given id of [databricks_metastore](../resources/metastore.md) object, that was created by Terraform or manually, so that special handling could be applied.
+
+-> **Note** If you have a fully automated setup with workspaces created by [databricks_mws_workspaces](../resources/mws_workspaces.md) or [azurerm_databricks_workspace](https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs/resources/databricks_workspace), please make sure to add [depends_on attribute](../index.md#data-resources-and-authentication-is-not-configured-errors) in order to prevent _authentication is not configured for provider_ errors.
+
+## Example Usage
+
+MetastoreInfo response for a given metastore id
+
+```hcl
+
+resource "databricks_metastore" "this" {
+  provider      = databricks.workspace
+  name          = "primary"
+  storage_root  = "s3://${aws_s3_bucket.metastore.id}/metastore"
+  owner         = var.unity_admin_group
+  force_destroy = true
+}
+
+data "databricks_metastore" "this" {
+  id = databricks_metastore.this.id
+}
+
+output "some_metastore" {
+  value = data.databricks_metastore.this.metastore_info[0]
+}
```

Comment:
We have this in other places as well: https://github.com/databricks/terraform-provider-databricks/blob/master/docs/data-sources/job.md?plain=1#L20. Have brought this up internally, not sure if there already exists a ticket for this.

Reply:
to be more precise, `metastore_info` is rendered as a list by the Terraform provider, so we need to use [0] to retrieve